src/components/pages/Theme/Theme.jsx

```diff
@@ -46,6 +49,209 @@ export const Theme = () => {
   const [isModalOpen, setIsModalOpen] = React.useState(false);
 
   // vars
+  const data = [
+    // missing workphone should be rendered as blank
+    {
+      name: 'John Doe',
+      govtID: '823456789',
+      homePhone: '3431231233',
+      workPhone: '',
```

Comment:
So it looks like the name of the keys have an effect on the table. This `workPhone` key seems to add the phone icon. You're not controlling that anywhere, right?

Reply:
We are! There is actually a `formatter` prop we can use:

```
formatter: (value) => (
        <div>
          <Icon name='phone' />
          {value &&
            `(${value.substring(0, 3)})-${value.substring(
              3,
              6
            )}-${value.substring(6)}`}
        </div>
      ),
```

Reply:
I left it there so it's an example of a custom cell, as we have in the original component library `Table`

Reply:
Ah, ok, I see it now.